Sources/SwiftLayout/Extension/Anchors/Anchors+Convenience.swift

```diff
@@ -176,4 +176,27 @@ extension Anchors {
         let height = Anchors(.height).equalTo(constant: size.height)
         return width.union(height)
     }
+    
+    /// ``Anchors`` for center to superview
+    /// - Parameters:
+    ///   - offsetX: ``CGFloat`` type. == superview.center.x += offsetX
+    ///   - offsetY: ``CGFloat`` type. == superview.center.y += offsetY
+    /// - Returns: ``Anchors``
+    public static func center<I: ConstraintableItem>(_ toItem:I, offsetX: CGFloat = .zero, offsetY: CGFloat = .zero) -> Anchors {
+        let x = Anchors(.centerX).equalTo(constant: offsetX)
+        let y = Anchors(.centerY).equalTo(constant: offsetY)
+        return x.union(y)
+    }
+    
+    /// ``Anchors`` for center to item
+    /// - Parameters:
+    ///   - toItem: ``ConstraintableItem`` type,  second item for constraint
+    ///   - offsetX: ``CGFloat`` type. == superview.center.x += offsetX
+    ///   - offsetY: ``CGFloat`` type. == superview.center.y += offsetY
+    /// - Returns: ``Anchors``
+    public static func center<I: ConstraintableItem>(_ toItem:I, offsetX: CGFloat = .zero, offsetY: CGFloat = .zero) -> Anchors {
+        let x = Anchors(.centerX).equalTo(constant: offsetX)
+        let y = Anchors(.centerY).equalTo(constant: offsetY)
```

Comment:
Is this the correct implementation?
It seems different from the description.

Reply:
human error human error